### ISSUE TITLE
Error message when launching FoxUnit a second time for the same folder

### ISSUE THREAD

**Issue by cwollenhaupt** [VFPX/FoxUnit#6]
When one instance is open and has executed a unit tests, then running FoxUnit in the same folder in a second VFP instance displays the following error message:

![image](https://user-images.githubusercontent.com/33864363/33658354-b3fc08d4-da7c-11e7-9c4e-fbc7debb55ed.png)

**Comment by eselje** [comment]
This reminds me of the old joke:
Patient: "Doctor, it hurts when I do this."
Doctor: "Well then don't do that!"

But...I suppose you must have a reason for doing this so it should be
fixed. :)

E

**Comment by cwollenhaupt** [comment]
True... I encountered this when I wanted to test the other issue. To check whether the problem was that VFP wasn't recompiling the PRG when it should (PRG is newer than FXP) or whether it was a cache issue, I tried running the same unit tests in a second instance that did not have the FXP in the cache.
